src/components/radios/__tests__/Radios.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Radios from '../Radios';
+
+describe('Radios', () => {
+  it('matches snapshot', () => {
+    const element = mount(
+      <Radios id="example" name="example">
+        <Radios.Radio id="example-1" value="yes">
+          Yes
+        </Radios.Radio>
+        <Radios.Radio id="example-2" value="no">
+          No
+        </Radios.Radio>
+      </Radios>,
+    );
+    expect(element).toMatchSnapshot();
+    element.unmount();
+  });
+
+  it('does not render the conditional content if checked is false', () => {
+    const element = mount(
+      <Radios id="example" name="example">
+        <Radios.Radio id="example-1" value="yes" checked={false} conditional={<p>Test</p>}>
+          Yes
+        </Radios.Radio>
+        <Radios.Radio id="example-2" value="no">
+          No
+        </Radios.Radio>
+      </Radios>,
+    );
+    element.find('input#example-1').simulate('change');
+    expect(element).toMatchSnapshot();
```

Comment:
It might be more appropriate to use a selector here rather than a snapshot test to assert that the conditional content is not being displayed

Reply:
Good point, will do 👍

Reply:
Done, thanks